source/WebConfig/cosa_webconfig_internal.c

```diff
@@ -305,6 +305,14 @@ int getWebConfigParameterValues(char **parameterNames, int paramCount, int *val_
 		                        paramVal[k]->type = ccsp_string;
 		                        k++;
 				}
+				else
+				{
+					WebcfgDebug("Webpa get : Get_Webconfig_URL %s is null\n", valuestr);
+					paramVal[k]->parameterName = strndup(WEBCONFIG_PARAM_URL, MAX_PARAMETERNAME_LEN);
```

Comment:
Instead of duplicating same code for parameterName and type, it can be added as common when value is empty or non-empty, similar to other parameters. Change the same for Data parameter too.